packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5890,47 +5894,134 @@ class MouseRegion extends SingleChildRenderObjectWidget {
   ///    internally implemented.
   final PointerEnterEventListener onEnter;
 
-  /// Called when a mouse pointer changes position without buttons pressed, and
-  /// the new position is within the region defined by this widget.
+  /// Called when a mouse pointer moves within this widget without buttons
+  /// pressed.
   ///
-  /// This callback is triggered when:
+  /// This callback is not triggered when the [MouseRegion] has moved
+  /// while being hovered by the mouse pointer.
   ///
-  ///  * An annotation that did not contain the pointer has moved to under a
-  ///    pointer that has no buttons pressed.
-  ///  * A pointer has moved onto, or moved within an annotation without buttons
-  ///    pressed.
-  ///
-  /// This callback is not triggered when
-  ///
-  ///  * An annotation that is containing the pointer has moved, and still
-  ///    contains the pointer.
+  /// {@macro flutter.mouseRegion.triggerTime}
   final PointerHoverEventListener onHover;
 
-  /// Called when a mouse pointer, with or without buttons pressed, has exited
-  /// this widget when the widget is still mounted.
+  /// Called when a mouse pointer has exited this widget when the widget is
+  /// still mounted.
   ///
-  /// This callback is triggered when the pointer has stopped to be contained
-  /// by the region of this widget, except when it's caused by the removal of
-  /// this widget. More specifically, the callback is triggered by
-  /// the following cases:
+  /// This callback is triggered when the pointer, with or without buttons
+  /// pressed, has stopped being contained by the region of this widget, except
+  /// when the exit is caused by the disappearance of this widget. More
+  /// specifically, this callback is triggered by the following cases:
   ///
-  ///  * This widget, which used to contain a pointer, has moved away.
-  ///  * A pointer that used to be within this widget has been removed.
-  ///  * A pointer that used to be within this widget has moved away.
+  ///  * This widget, which is being hovered by a pointer, has moved away.
+  ///  * A pointer that is hovering this widget has been removed.
+  ///  * A pointer that is hovering this widget has moved away.
   ///
   /// And is __not__ triggered by the following case,
```

Comment:
You could just eliminate the bullet (since there's only one now), and say "`And is __not__ triggered when this widget disappears while being hovered over.`"

Reply:
The reason for the bullet is to make the 4 cases aligned, so that it's obvious that all cases have been enumerated. I still prefer keeping the bullets.